### PR TITLE
dd: update translations

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Italian part is approved, i'll leave the rest to other codeowners since i don't know those languages

**Comment by Magrid0** [comment]
> The Italian page is outdated

oh my bad I didn't notice

> Is that something you can pick up? We can do this in PR or in a separate PR

Sure, I'll work on it immediatly

File: pages.zh/osx/dd.md [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
 `dd bs=4m conv=noerror if={{/dev/源设备}} of={{/dev/目标设备}}`
 
-- 使用内核随机驱动程序生成 100 个随机字节的文件：
+- 使用内核随机驱动程序生成 ... 个随机字节的文件：
```
Comment (blueskyson) [inline pages.zh/osx/dd.md:14]:
```suggestion
- 使用内核随机驱动程序生成指定数量个随机字节的文件：
```

[the remark this answers is not in the document]
Comment (blueskyson) [reply pages.zh/osx/dd.md:14]:
```suggestion
- 使用内核随机驱动程序生成指定数量个随机字节的文件：
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**